packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -104,6 +131,192 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         }
     }
 
+
+    private async _onCancel() {
+        if (this._shippingAddress) {
```

Comment:
Is there any varian to get shippingAddress to avoid using this._shippingAddress variable?

Reply:
As far as I understand, you can get shipping address from consignments.

Reply:
fixed